src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx

```diff
@@ -9,10 +9,10 @@ import { K8SResourceListType } from '../Types'
 import ResourceListEmptyState from './ResourceListEmptyState'
 import ReactSelect from 'react-select'
 import { Option } from '../../../components/v2/common/ReactSelect.utils'
-import '../ResourceBrowser.scss'
 import AppDetailsStore from '../../v2/appDetails/appDetails.store'
 import { toast } from 'react-toastify'
 import { EventList } from './EventList'
+import '../ResourceBrowser.scss'
```

Comment:
Are we utilising this scss in this file?

Reply:
@shivani170 As discussed, removed the changes from this file from PR. Need to confirm before making any changes to this file.